.travis.yml

```diff
@@ -23,18 +23,18 @@ jobs:
       script:
         - "vendor/bin/phpunit --verbose"
 
-    - name: "WP 6.0 - PHP 5.6"
+    - name: "WP 6.1 - PHP 5.6"
```

Comment:
Please don't modify the Travis config to avoid merge conflicts. This is already done on master branch.

Reply:
Yup, done in 811fe4d